packages/zoe/src/typeGuards.js

```diff
@@ -1,20 +1,67 @@
 // @ts-check
 
-import { AmountShape } from '@agoric/ertp';
+import {
+  AmountShape,
+  AssetKindShape,
+  DisplayInfoShape,
+  IssuerShape,
+  IssuerKitShape,
+  BrandShape,
+  PaymentShape,
+} from '@agoric/ertp';
 import { M } from '@agoric/store';
 import { TimestampValueShape } from '@agoric/swingset-vat/src/vats/timer/typeGuards.js';
 
+export const HandleI = M.interface('Handle', {});
+
+// keywords have an initial cap
+export const KeywordShape = M.string();
+
+// TODO should be exported from Notifiers.
+export const NotifierShape = M.any();
```

Comment:
Shouldn't this be

```suggestion
export const NotifierShape = M.remotable('Notifier');
```

I agree it should be in the notifier package. This PR may as well do that, rather than first introducing it in the wrong place.

Reply:
Done